code/game/objects/items/devices/scanners.dm

```diff
@@ -49,17 +49,17 @@ BREATH ANALYZER
 	var/degree = "none"
 
 	switch(damage_ratio)
-		if(0.001 to 0.1)
+		if(0 to 5)
```

Comment:
So I'm hugely not a fan of swapping anything from float to integer, they genuinely aren't as bad as your PR implies they are, and floats should generally always be preferred over integers for the improved precision (Yes, floats have *substantially more* precision than integers, not less). But beyond that, these aren't even the same ratios anymore. I don't see you replacing any other damage values anywhere else, yet these don't actually line up with the original number balancing.

Reply:
After some consideration, the description of a wound being "lethal" basically confirms to me that these values are pre-brainmed. I have ported the values straight from baystation as a result.